native/src/components/MapView.tsx

```diff
@@ -79,7 +93,8 @@ const MapView = ({ boundingBox, featureCollection, currentFeature }: MapViewProp
 
   const requestPermission = useCallback(() => {
     requestLocationPermission().then(onLocationPermissionRequest)
-  }, [onLocationPermissionRequest])
+    locationPermissionGranted && setFollowUserLocation(!followUserLocation)
```

Comment:
What is happening here? Can you add a comment please?

Reply:
done

Reply:
I think it should not change `followUserLocation` to the opposite but instead always to true.